config/passport/passport.github.config.js

```diff
@@ -0,0 +1,75 @@
+// Package imports
+const GitHubStrategy = require('passport-github2').Strategy;
+const env = require('dotenv').config();
+const to = require('await-to-js').default;
+
+// Constant imports
+const globalConstants = require('../../constants/constants');
```

Comment:
```suggestion
const globalConstants = require('../../constants/constants');
```
Rename it to constants

```suggestion
const configConstants = require('../constants');
```
Rename it to gitHubConstants

Please use meaningful names

Reply:
Done